services/minespace-web/src/selectors/varianceSelectors.js

```diff
@@ -79,3 +80,17 @@ export const getVarianceStatusOptionsHash = createSelector(
   [getDropdownVarianceStatusOptions],
   createLabelHash
 );
+
+export const getDropdownVarianceDocumentCategoryOptions = createSelector(
+  [getVarianceDocumentCategoryOptions],
+  (options) =>
+    options.map((option) => {
+      const composedLabel = `${option.description} Document`;
+      return { value: option.variance_document_category_code, label: composedLabel };
+    })
+);
+
+export const getVarianceDocumentCategoryOptionsHash = createSelector(
+  [getDropdownVarianceDocumentCategoryOptions],
+  createLabelHash
+);
```

Comment:
I have some big concerns about code duplication and maintainability issues. Will chat in person.

Reply:
Discussed in person during standup.